backend/alembic/versions/967305fbe7aa_backend_app.py

```diff
@@ -0,0 +1,38 @@
+"""backend_app
+
+Revision ID: 967305fbe7aa
+Revises: fdbceb410b8d
+Create Date: 2023-06-19 15:56:43.145325
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '967305fbe7aa'
+down_revision = '1a9d5e4a1719'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('backendapp',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('app_id', sa.String(), nullable=False),
+    sa.Column('owner', sa.Integer(), nullable=False),
```

Comment:
This will cause users with apps not being able to delete their accounts. We probably need a workflow for that. I don't think we want to remove the app in that case?

Reply:
I'd assume we don't want an app to have no owner, and that if you're the last developer then you have to transfer ownership to a new maintainer or EOL the app before you can delete your account.